mappings/net/minecraft/network/packet/s2c/play/ActionBarTextSetS2CPacket.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_5894 net/minecraft/network/packet/s2c/play/ActionBarTextSetS2CPacket
```

Comment:
Don't we call this "hotbar message"?

Reply:
I don't think so, and if we do then that would be wrong. Action Bar is the name given by the /title command.

Reply:
hotbar message is what we have in in game hud